apps/explorer/priv/repo/migrations/20240205141332_reserved_addresses_table.exs

```diff
@@ -0,0 +1,24 @@
+defmodule Explorer.Repo.Migrations.ReservedAddressesTable do
+  use Ecto.Migration
+  @disable_ddl_transaction true
+  @disable_migration_lock true
+
+  def change do
```

Comment:
A migration script should either have 1) both `up` and `down` functions or 2) a `change` function.

According to documentation, the [`change` function ](https://hexdocs.pm/ecto_sql/Ecto.Migration.html#module-change) can be used instead of `up`/`down`, though not all commands are reversible.

`def change` should be changed to `def up` in all the migrations here, and check that the migration & rollback still work

Reply:
ok agree, method name updated for every script.
I also perform a local test with success activating the native contract visualization and rollback to old version. I also reactivate again the native contract feature with success as well.